src/stylus/fablevision/location-indicator.styl

```diff
@@ -0,0 +1,25 @@
+.geniblocks.location-indicator
+  width: 1096px
+  height: 93px
+  background-image: url('../resources/fablevision/huds/Location_bar.png')
+  background-size: 100% 100%
+
+  .hud-text-area
+    position: relative
+    left: 90px
+    top: 16px
+    width: 770px
+    display: flex
+    flex-direction: column
+    justify-content: center
+    align-items: center
+    font-family: 'Yanone Kaffeesatz'
```

Comment:
Yes, I think once we get past this classroom test we should give some thought to better organization of the CSS. I think it's useful to have a few instances first so we have a better idea of what the general case looks like.